tests/test_cadquery.py

```diff
@@ -2176,3 +2182,74 @@ def test_assembleEdges(self):
             edge_wire = edge_wire.add(Workplane(plane_list[i+1]).workplane(offset=-offset_list[i+1]).spline(edge_points[i+1]))
         edge_wire = [o.vals()[0] for o in edge_wire.all()]
         edge_wire = Wire.assembleEdges(edge_wire)
+        self.assertTrue(edge_wire.isValid())
+        self.assertEqual(len(edge_wire.Wires()), 1)
+        
+    def test_interpPlate(self):
+        
+        decimal_places = 0
+        
+        # example from PythonOCC core_geometry_geomplate.py, use of thickness = 0 returns 2D surface.
+        thickness = 0
+        edge_points = [[0.,0.,0.], [0.,10.,0.], [0.,10.,10.], [0.,0.,10.]]
+        surface_points = [[5.,5.,5.]]
+        plate_0 = Workplane('XY').interpPlate(edge_points, surface_points, thickness)
+        self.assertTrue(plate_0.val().isValid())
+        self.assertAlmostEqual(plate_0.val().Area(), 141.218823892, decimal_places)
```

Comment:
Out of curiosity: how did you calculate the "correct" area?

Reply:
A comment in the source code would probably be enough to explain how to calculate it manually.

Reply:
The numerical values are the result of ``plate_0.val().Area()`` on my own machine, under Linux.
I don't know if it actually makes sense to use this as a test or if I should come with something else. Any thoughts on this?

Reply:
What else would you use other than area? The nice thing about area is that it's a single metric that tells you a lot about whether complex geometry was created properly or not.

Reply:
I have nothing against area - I was just curious about the reference value.

Reply:
@bragostin maybe indeed add a comment that it was computed numerically and that's it.

Reply:
OK, I added this comment in the docstring of the test: ``Tests the interpPlate() functionnalites.      Numerical values of Areas and Volumes were obtained with the Area() and Volume() functions on a Linux machine under Debian 10 with python 3.7.``

Fine tuning the number of decimal places for each test now yields this:
```python
self.assertAlmostEqual(plate_0.val().Area(), 141.218823892, 1)
self.assertAlmostEqual(plate_1.val().Volume(), 26.124970206, 3)
self.assertAlmostEqual(plate_2.val().Volume(), 10.956054314, 0)
self.assertAlmostEqual(plate_3.val().Volume(), 0.459661032, 1)
self.assertAlmostEqual(plate_4.val().Volume(), 7.760559490, 3)
```